apps/meteor/app/apps/server/communication/rest.js

```diff
@@ -362,7 +362,7 @@ export class AppsRestApi {
 
 					let result;
 					try {
-						result = HTTP.get(`${baseUrl}/v1/apps/featured`, {
+						result = HTTP.get(`${baseUrl}/v1/featured-apps`, {
```

Comment:
This is the URL in the Marketplace API, not the Rocket.Chat server. Please double check if the Marketplace API has introduced this new endpoint, as my test with cURL (`curl https://marketplace.rocket.chat/v1/featured-apps`) hints it doesn't exist

Reply:
This endpoint is still in the marketplace staging, it will be released shortly.

Reply:
This is the staging URL if you want to test it: https://marketplace.staging.cloud.rocket.chat/v1/featured-apps